yelyzaveta.kozhevnikova/task1.rb

```diff
@@ -0,0 +1,11 @@
+arr = Array.new(3) { Array.new(3) { rand(-10..10) } }
+pp(arr)
+res = 0
+for x in 0..2
```

Comment:
regular for loops is not a case for ruby development. In ruby, we use iterators(e.g each, map, select, count) to implement cycled operation.
For example:
```
arr.each do |element|
  puts element
end
```
#flatten method allows you not going through every nested loop
#count method can be used to calculate the size of an array by a given condition.

Reply:
Thanks)

arr = Array.new(3) { Array.new(3) { rand(-10..10) } }
pp(arr)
res = 0
(0..2).each do |x| 
  (0..2).each do |y| 
    i = arr[x].at(y)
    puts(i) if i.positive?
    res += i if i.positive?
  end
end
pp(res)